TUM-Dev/NavigaTUM
# chore:tailwind

## Proposed Changes (include Screenshots if possible)

- This PR migrates a part of the frontend to tailwind as a cleaner theming is blocking #473 

## How to test this PR

1. Launch the webclient
2. see that the design looks the same

## How has this been tested?

- TODO set up visual regression testing

## Checklist
